do_snowDA.sh

```diff
@@ -57,7 +59,7 @@ B=30  # back ground error std.
 
 SAVE_IMS="YES" # "YES" to save processed IMS IODA file
 SAVE_INCR="YES" # "YES" to save increment (add others?) JEDI output
-SAVE_TILE="NO" # "YES" to save background in tile space
+SAVE_TILE="YES" # "YES" to save background in tile space
 
 THISDATE=${THISDATE:-"2015090118"}
 
```

Comment:
None of the variables above this line should be changed in your PR.

Reply:
Changed back